unicodetools/src/main/java/org/unicode/props/UnicodeProperty.java

```diff
@@ -459,6 +459,19 @@ public UnicodeSet getSet(PatternMatcher matcher, UnicodeSet result) {
         List<String> valueAliases = new ArrayList<>(1); // to avoid reallocating...
         List<String> partAliases = new ArrayList<>(1);
         UnicodeMap<String> um = getUnicodeMap_internal();
+        if (matcher.test(null)) {
+            int previousNullStart = 0;
+            for (var range : um.entryRanges()) {
+                result.addAll(previousNullStart, range.codepoint - 1);
+                previousNullStart = range.codepointEnd + 1;
+            }
+            result.addAll(previousNullStart, 0x10FFFF);
+        }
+        if (matcher == UnicodeProperty.NULL_MATCHER) {
```

Comment:
We could put this inside the previous block, since it will only test = if the matcher.test(null), right?

Reply:
Done.